src/Agent/NewRelic/Profiler/Profiler/CorProfilerCallbackImpl.h

```diff
@@ -306,18 +298,6 @@ namespace NewRelic { namespace Profiler {
             return S_OK;
         }
 
-
-        virtual DWORD OverrideEventMask(DWORD eventMask)
-        {
-#ifndef PAL_STDCPP_COMPAT
-            if (!_isCoreClr)
-            {
-                _moduleInjector.reset(new ModuleInjector::ModuleInjector());
```

Comment:
I'm not sure I follow what this was accomplishing before, but it's no longer needed?

Reply:
It created an instance of a class that didn't have any instance level methods or data, so there was no state that needed to be managed for that class which means that we were replacing the current instance of the injector with a new one without a reason to do that. That class only contains static methods, and with this change the usage of those methods were updated to use a static invocation instead of an instance-level invocation style. More specifically `_moduleInjector->InjectIntoModule(*module)` was changed to `ModuleInjector::ModuleInjector::InjectIntoModule(*module, _isCoreClr)` because InjectIntoModule was a static method and we do not need an instance of ModuleInjector to invoke it.  The previous code is probably a remnant of some older behavior that has not been in place for a while.